jobs/openshift4/cluster/create/openshift4-cluster-create.yaml

```diff
@@ -0,0 +1,68 @@
+---
+- job:
+    name: openshift4-cluster-create
+    display-name: 'OpenShift 4 Cluster Create'
+    project-type: pipeline
+    concurrent: true
+    parameters:
+      - string:
+          name: 'CLUSTER_NAME'
+          default: ''
+          description: 'Name of the cluster (e.g. your kerberos username). Defaults to build number'
+      - choice:
+          name: 'COMPUTE_REPLICAS'
+          choices:
+            - '2'
+            - '3'
+          description: 'Number of compute replicas for the Openshift 4 cluster'
+          default: '2'
+      - choice:
+          name: 'CONTROL_PLANE_REPLICAS'
+          choices:
+            - '1'
+            - '2'
+            - '3'
+          description: 'Number of control plane replicas for the Openshift 4 cluster'
+          default: '1'
+      - string:
+          name: 'RECIPIENTS'
+          default: integreatly-qe@redhat.com
+          description: "Whitespace- or comma-separated list of email addresses to which the result of this pipeline execution will be sent"
```

Comment:
Can this be left blank? Was this asked for by someone?

Reply:
Since this pipeline will also be executed nightly I would see some value in sending the email with the results to the subscribers. I can add logic to skip sending such email if the list is empty. Or do you want this param to be removed completely?

Reply:
We definitely want some results from the nightly pipelines and I think this mailing list makes perfect sense so I would keep it as it is.

Reply:
default value changed to blank